android/README.md

```diff
@@ -0,0 +1,19 @@
+# Android client application
+
+Android app is written with Kotlin language as the client endpoint for EatersLab system. App allows users to search for nearby or favourite canteens and check occupancy data and other available information for cafeterias.
+App is designed for min SDK 21 version so Android Lollipop or newer devices are supported (so really few devices are unsupported)
+
+## App building instructions
+
+App is developed in AndroidStudio and for development should be imported as standard Android app project. Before building the project and indexing files in IDE the generation process is needed.
+
+### Client model generation
```

Comment:
I would use "API" in this heading so it's easier to find for people who look for API generation.

Reply:
fixed